src/device-registry/config/constants.js

```diff
@@ -52,9 +52,9 @@ const stageConfig = {
 };
 
 const defaultConfig = {
-  NETWORKS: process.env.NETWORKS.split(","),
-  ACTIVITY_TYPES: process.env.ACTIVITY_TYPES.split(","),
-  MAINTENANCE_TYPES: process.env.MAINTENANCE_TYPES.split(","),
+  NETWORKS: (process.env.NETWORKS).split(","),
+  ACTIVITY_TYPES: (process.env.ACTIVITY_TYPES).split(","),
+  MAINTENANCE_TYPES: (process.env.MAINTENANCE_TYPES).split(","),
```

Comment:
Do these brackets resolve anything? @BenjaminSsempala

Reply:
Oh please ignore those, I just forgot to remove them after getting the envs